git/commands.md

```diff
@@ -111,6 +113,7 @@ To create branch You should run
 
 ```
 git branch <name_branch>
+git branch vadim
```

Comment:
Please don't change this line. I suppose the students of younger courses wouldn't appreciate their branches all named `vadim`